.circleci/config.yml

```diff
@@ -0,0 +1,94 @@
+version: 2
+defaults: &defaults
```

Comment:
Found this pattern after investigating CircleCI configs on other projects. Reduces the repetition and noise in the environment setup of jobs.